### PR TITLE
Redesign chat example

### PR DESCRIPTION
**Pull request by SebastianAigner**
New look:

IMG_2137

### CODE REVIEW

**Comment by SebastianAigner** [comment]
Seems to suffer from #2770 on Android – any help / input on that appreciated.

File: examples/chat/iosApp/Configuration/Config.xcconfig
Comment (dima-avdeev-jb) [inline examples/chat/iosApp/Configuration/Config.xcconfig:1]: I think it is better to leave TEAM_ID empty

**Comment by dima-avdeev-jb** [comment]
Very cool! Really like the new look!

File: examples/chat/shared/src/commonMain/resources/background.png
Comment (dima-avdeev-jb) [reply examples/chat/shared/src/commonMain/resources/background.png:1]: I think for now we can use static image as a background.
But it is a really good suggestion for the future. Maybe someone of us can implement this logic as a dogfooding project.

File: examples/chat/shared/src/commonMain/resources/stock1.png
Comment (dima-avdeev-jb) [reply examples/chat/shared/src/commonMain/resources/stock1.png:1]: Yeah, this file is really big. But at first we can convert them to **JPEG**. And because these pictures are used as a miniature - we can change size, for example, to 256px

Comment (SebastianAigner) [reply examples/chat/shared/src/commonMain/resources/stock1.png:1]: Stock images have been resized and JPEGified. They're now <20KiB each.

File: examples/chat/shared/src/commonMain/resources/background.png
Comment (SebastianAigner) [reply examples/chat/shared/src/commonMain/resources/background.png:1]: Background is now also a JPG, weights 480 KiB. Usually, pictures with gradients should probably stay PNGs for reasons of color-banding, but it seems that at this resolution, I can't optically spot any compression artifacts yet.

I'd love for it to be an XML, but Android XMLs don't support blend modes, which makes the image look super dirty:

Screenshot 2023-05-16 at 13 51 09

File: examples/chat/iosApp/Configuration/Config.xcconfig
Comment (SebastianAigner) [reply examples/chat/iosApp/Configuration/Config.xcconfig]: You're of course entirely correct – somehow this one always ends up in my commits by accident. 😱

File: examples/chat/shared/src/commonMain/resources/background.png
Comment (SebastianAigner) [reply examples/chat/shared/src/commonMain/resources/background.png:1]: While dynamic generation would might be fun, it seems like it'd be very much out of scope for what we're trying to accomplish 😅

Comment (dima-avdeev-jb) [reply examples/chat/shared/src/commonMain/resources/background.png:1]: Yeah, we will make generator by ourselves in dogfooding mode

**Comment by dima-avdeev-jb** [comment]
Also file resources/background.pxd is redundant i think

**Comment by SebastianAigner** [comment]
> Also file resources/background.pxd is redundant I think

Well, it's the actual vectory file, with layers, as saved by Pixelmator. Since I don't want to recreate the whole image from scratch on every change, I'd rather have it in here 😅

**Comment by SebastianAigner** [comment]
With the font sizes fixed to `sp`, it now also looks good on Android. Ready to merge from my perspective, unless y'all still have any changes?

Screenshot_20230516_211945

**Review by dima-avdeev-jb — APPROVED** [review]

**Comment by dima-avdeev-jb** [comment]
Can you please remove **resources/background.pxd** after completion of this PR and before merge?

**Review by MatkovIvan — APPROVED** [review]

**Comment by SebastianAigner** [comment]
Adding this here in case we need the file in the future.
[background.pxd.zip](https://github.com/JetBrains/compose-multiplatform/files/11523428/background.pxd.zip)

**Pull request merged**
